Directory.Build.props

```diff
@@ -0,0 +1,52 @@
+<Project>
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
```

Comment:
This is what breaks the x64 build. The 32 bit build is created first and some of its intermediate state is probably leaking into the 64 bit build.

Lets just remove this line to bring things back to how they currently are on bleed.